cli/build_vfs.go

```diff
@@ -0,0 +1,7 @@
+// +build vfs
```

Comment:
Use `go_vfs` as the build constraint name and rename this file to `build_go-vfs.go`

Reply:
Done